hack/istio/install-bookinfo-demo.sh

```diff
@@ -140,6 +140,18 @@ else
   exit 1
 fi
 
+IS_OPENSHIFT="false"
+IS_MAISTRA="false"
+if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  IS_OPENSHIFT="true"
+  if [[ $([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true") ]]; then
+    IS_MAISTRA="true"
```

Comment:
that's alot of extra stuff you don't need (that `&& echo "true"` is superfluous the way you have it). If you really want the value to be false as opposed to empty string when not maistra, then I suggest:
```suggestion
  IS_MAISTRA=$([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true" || echo "false")
```